main.go

```diff
@@ -289,6 +291,23 @@ func (cfg *config) doProxy(w http.ResponseWriter, r *http.Request) {
 	h.ServeHTTP(w, r)
 }
 
+func (cfg *config) listModules(w http.ResponseWriter, r *http.Request) {
+	log.Debugf("Listing modules")
+	w.Header().Set("Content-Type", "text/html; charset=utf-8")
+	tmpl, err := template.New("modules").Parse(`
+		<h2>Exporters:</h2>
+			<ul>
+				{{range $name, $cfg := .Modules}}
+					<li><a href="/proxy?module={{$name}}">{{$name}}</a></li>
+				{{end}}
+			</ul>`)
+	if err != nil {
```

Comment:
This should never fail, so I'm tempted to say you can just _ out the err above. Failing that, if we check the error here, then please http.Error(....) and return (you are currently exeuting the template even if this fails).